libcloud/compute/drivers/gce.py

```diff
@@ -1733,15 +1733,19 @@ class GCENodeDriver(NodeDriver):
 
     IMAGE_PROJECTS = {
         "centos-cloud": ["centos"],
+        "cos-cloud": ["container-vm"],
         "coreos-cloud": ["coreos"],
         "debian-cloud": ["debian", "backports"],
         "gce-nvme": ["nvme-backports"],
         "google-containers": ["container-vm"],
         "opensuse-cloud": ["opensuse"],
         "rhel-cloud": ["rhel"],
         "suse-cloud": ["sles", "suse"],
+        "suse-byos-cloud": ["sles", "suse"],
```

Comment:
The list may have been updated since you submitted the pull request, but I don't see suse-byos-cloud in https://cloud.google.com/compute/docs/images#os-compute-support. Do you think it's still needed?

Reply:
Yes, it is still needed. It's not in the list however in the SUSE details below it's mentioned:

> If you need to reuse your existing SLES licenses to create Compute Engine instances, you can use Bring Your Own Subscription (BYOS) images available in the suse-byos-cloud project.